src/NServiceBus.RabbitMQ/Routing/ISupportNativeDelays.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using global::RabbitMQ.Client;
+
+    /// <summary>
+    /// An extension for <see cref="IRoutingTopology"/> implementations which allows
+    /// for receiving messages from the delay infrastructure.
```

Comment:
👍 to `ISupportDelayedDelivery`. Perhaps some tweaks to the XML docs may also result from the rename.